src/app/patient-table/patient-table.component.css

```diff
@@ -0,0 +1,42 @@
+.example-container {
+    position: relative;
+}
+
+.example-table-container {
+    position: relative;
+    min-height: 200px;
+    max-height: 400px;
+    overflow: auto;
+}
+
+table {
+    width: 100%;
+}
+
+.example-loading-shade {
+    position: absolute;
+    top: 0;
+    left: 0;
+    bottom: 56px;
+    right: 0;
+    background: rgba(0, 0, 0, 0.15);
+    z-index: 1;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+}
```

Comment:
I would remove position absolute and try to make a loader to appear as a part of the table, not somewhere on the top of the page, maybe something like this:


    background: rgba(0, 0, 0, 0.15);
    display: flex;
    align-items: center;
    justify-content: center;
    height: 30vh;



and move the spinner in html after the table, something like this:
![image](https://user-images.githubusercontent.com/11686968/220999754-9b087fb7-4e4a-4b75-8ea8-a37ba84b9d1a.png)

will get us following result:
![image](https://user-images.githubusercontent.com/11686968/220999841-69ae20fd-0975-45ca-adc2-4259df69810c.png)

Reply:
for me, what I posted on discord is cool

Reply:
look, remove properties form css that are not used:
![image](https://user-images.githubusercontent.com/11686968/221157165-24538c3b-ba29-404e-976b-414066f9637d.png)

`top`, `left`, `bottom`, `right` only make sense with position absolute, and add `height: 30vh` please so it has some spcace

Reply:
done